po/he.po

```diff
@@ -8747,7 +8747,7 @@ msgstr "חשבונאות לאנשים פרטיים ועסקים קטנים."
 #: gnucash/gnome-utils/gnc-main-window.c:4736
 msgid "translator-credits"
 msgstr ""
-"אבי מרקוביץ 2021, 2020, 2019 avi.markovitz@gmail.com\n"
+"אבי מרקוביץ avi.markovitz@gmail.com 2021, 2020, 2019\n"
 "אורי הוך 2006"
```

Comment:
@avma How about 

1. `2019-2021` or similar.
2. Ori's line ine the same format?